schema/validator.go

```diff
@@ -41,7 +41,7 @@ func (e ValidationError) Error() string {
 func (v Validator) Validate(src io.Reader) error {
 	buf, err := ioutil.ReadAll(src)
 	if err != nil {
-		return errors.Wrap(err, "unable to read manifest")
+		return errors.Wrap(err, "unable to read the target file")
```

Comment:
Runtime-spec calls this the [“document”][1].  I think we probably want to use consistent wording (although I don't have a strong preference between “document” and “ttarget”).

[1]: https://github.com/opencontainers/runtime-spec/blob/22c29c18eddc75ccee9f6b202f61524f29653c5f/schema/validate.go#L15